components/pipedrive/actions/add-activity/add-activity.mjs

```diff
@@ -1,133 +1,154 @@
-// legacy_hash_id: a_k6iKYA
-import { axios } from "@pipedream/platform";
+import pipedriveApp from "../../pipedrive.app.mjs";
+import utils from "../../common/utils.mjs";
 
 export default {
   key: "pipedrive-add-activity",
   name: "Add Activity",
-  description: "Adds a new activity. Includes more_activities_scheduled_in_context property in response's additional_data which indicates whether there are more undone activities scheduled with the same deal, person or organization (depending on the supplied data).",
-  version: "0.1.1",
+  description: "Adds a new activity. Includes more_activities_scheduled_in_context property in response's additional_data which indicates whether there are more undone activities scheduled with the same deal, person or organization (depending on the supplied data). See the Pipedrive API docs for Activities [here](https://developers.pipedrive.com/docs/api/v1/#!/Activities). For info on [adding an activity in Pipedrive](https://pipedrive.readme.io/docs/adding-an-activity)",
```

Comment:
I think this would make a better doc link:
```suggestion
  description: "Adds a new activity. Includes more_activities_scheduled_in_context property in response's additional_data which indicates whether there are more undone activities scheduled with the same deal, person or organization (depending on the supplied data). See the Pipedrive API docs for Activities [here](https://developers.pipedrive.com/docs/api/v1/#!/Activities). For info on [adding an activity in Pipedrive](https://developers.pipedrive.com/docs/api/v1/Activities#addActivity)",
```